app/controllers/samples_controller.rb

```diff
@@ -389,11 +389,11 @@ def results_folder
     render template: "samples/folder"
   end
 
-  def fastqs_folder
-    @file_list = current_power.updatable_samples.include?(@sample) ? @sample.fastqs_folder_files : []
-    @file_path = "#{@sample.sample_path}/fastqs/"
-    render template: "samples/folder"
-  end
+  # def fastqs_folder
```

Comment:
I was just remove it.
We can go through history to recover if necessary.

Reply:
ok.